pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -7,11 +7,21 @@ SPDX-License-Identifier: Apache-2.0
 package didexchange
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
 	"testing"
 
-	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
-
+	"github.com/google/uuid"
 	mocktransport "github.com/hyperledger/aries-framework-go/pkg/internal/didcomm/transport/mock"
+	errors "golang.org/x/xerrors"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/leveldb"
```

Comment:
We should have a TODO for a mock storage provider. I find it a bit odd to keep seeing LevelDB as the mock provider :-).

Reply:
Created this to address: https://github.com/hyperledger/aries-framework-go/issues/202

Reply:
Note: a new `mockStore` was added in this PR at the end of this file (used in a couple of the tests)